server/websocket.go

```diff
@@ -650,6 +651,16 @@ func (p *Plugin) handleJoin(userID, connID, channelID, title, threadID string) (
 		}, &model.WebsocketBroadcast{ChannelId: channelID, ReliableClusterSend: true})
 	}
 
+	clientStateData, err := json.Marshal(state.Call.getClientState(p.getBotID(), userID))
+	if err != nil {
+		p.LogError("failed to marshal client state", "err", err.Error())
+	} else {
+		p.publishWebSocketEvent(wsEventCallState, map[string]interface{}{
```

Comment:
I realize I could have re-used `wsCallStart` by extending it but it seemed cleaner to have a new event, especially as we'll likely need to break/deprecate some before 1.0.

Reply:
It will arrive before because it's queued before. But it wouldn't really matter since the state would be the same and loaded on top. If anything we could skip it but it would be a breaking change so not going to do it here.

Reply:
Ah -- I mixed the two, I meant the other way around. If call state arrives before join -- would that be possible? If it is, would that break anything on the redux side?

Reply:
I don't think so, joining "twice" wouldn't do anything. But it also shouldn't be possible, events are queued and received in the order they are sent. We are also making sure to use the TCP channel intra-cluster so there's no way they'd get out of order.

Reply:
Cool, thanks!